src/DonorDashboards/Tabs/EditProfileTab/PasswordRoute.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Give\DonorDashboards\Tabs\EditProfileTab;
+
+use Give\DonorDashboards\Helpers\SanitizeProfileData as SanitizeHelper;
+use Give\DonorDashboards\Profile as Profile;
+use Give\DonorDashboards\Tabs\Contracts\Route as RouteAbstract;
+use WP_REST_Request;
+
+/**
+ * @since 2.10.0
```

Comment:
Same question about the tags from the other PR

Reply:
Good 'ol copy and pase 😎 
Resolved, see https://github.com/impress-org/givewp/pull/7126/commits/89fb91b87b567583eafc40f1b9e572bfb9a07d4d